src/Common/Common.service.ts

```diff
@@ -530,3 +536,46 @@ export const getGitBranchUrl = (gitUrl: string, branchName: string): string | nu
     else if (trimmedGitUrl.includes(GitProviderType.AZURE)) return `${trimmedGitUrl}/src/branch/${branchName}`
     return null
 }
+
+export const getGlobalVariables = async ({
+    appId,
+    isCD = false,
```

Comment:
Instead of isCD, shouldn't we have the stageType and pass it down the filter?

Reply:
copied code 
and this does not improve functionality, when we call this service from CD we pass isCD as true